core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2261,6 +2261,21 @@ private void checkLiteral2(String expression, String expected) {
         .ok(expectedOracle);
   }
 
+  @Test public void testPreserveAlias() {
+    final String sql = "select \"warehouse_class_id\" as \"id\", \"description\" "
+        + "from \"warehouse_class\"";
+    final String expected = "SELECT \"warehouse_class_id\" AS \"id\", \"description\"\n"
+        + "FROM \"foodmart\".\"warehouse_class\"";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testPreservePermutation() {
```

Comment:
This was already passing, but it is testing a related behavior we care about that I didn't see tested already